doc/Reference/Markup/StaticAndThemeResources.md

```diff
@@ -0,0 +1,156 @@
+---
+uid: Reference.Markup.StaticAndThemeResources
+---
+
+# Using Static and Theme Resources
+
+Using Static or Theme Resources is built on top of the `IDependencyPropertyBuilder<T>`. This allows the greatest degree of flexibility and extensibility.
+
+To get started let's assume that we have a string resource called `AppTitle` in our ResourceDictionary. This could be either on the element itself, somewhere within the Visual Tree or even within the Application Resources. In this case we might provide the value like:
+
+```csharp
+new TextBlock()
+    .Text(x => x.StaticResource("AppTitle"))
+```
+
+Similarly to this we might want to use a Theme Resource such as `ApplicationPageBackgroundThemeBrush` to set the Background of our Page. In this case we will see we can use nearly identical syntax like:
+
+```csharp
+public partial class MyPage : Page
+{
+    public MyPage()
+    {
+        this.Background(x => x.ThemeResource("ApplicationPageBackgroundThemeBrush"));
+    }
+}
+```
+
+## Using Strongly Typed Resources
+
+One of the great benefits of using C# Markup of course is the strong typing. There may be many times where we might simply be able to provide a class like:
+
+```csharp
+public static class MyResources
+{
+    public const string MyString = "Hello from C# Markup";
+}
+```
+
+This of course works great becuase we can bypass the overhead of looking up and applying a Resource that could be located anywhere in the Visual Tree from the element we are working on up to the Window Content or within the Application Resources. It simplifies our code to simply set the value like:
+
+```csharp
+new TextBlock()
+    .Text(MyResources.MyString)
+```
+
+Unfortunately app development is often not that simple. As a result it is helpful to find ways that we can make use of resources from our markup code. C# Markup provides us a few useful API's to achieve our goals.
+
+### Creating a new Resource
+
+Many times we may want to create resources, but we also want to be able to easily reference them later. Let's start by looking at a case where we need to define a path string that we will use later for an icon.
+
+```cs
+public static class MyResources
+{
+    public static readonly Resource<Geometry> MyIcon =
+        StaticResource.Create<Geometry>(nameof(MyIcon), "{Path String}");
+}
+```
+
+Looking at this sample we actually are achieving multiple things. One is that we are able to more easily make use of our path string as we can provide it the generic argument for a Geometry object which is what will be expected when we use it. We also are able to avoid trying to figure out how to convert it. Lastly in the case of WinUI we get the advantage that this will be created as a Geometry object each time we need it, this will keep it from being parented and prevent it from being reused later. 
```

Comment:
```suggestion
Looking at this sample we actually are achieving multiple things. One is that we are able to more easily make use of our path string as we can provide it the generic argument for a `Geometry` object which is what will be expected when we use it. We also are able to avoid trying to figure out how to convert it. 

Lastly, in the case of WinUI we get the advantage that this will be created as a `Geometry` object each time we need it, this will keep it from being parented and prevent it from being reused later. 
```